homeassistant/components/verisure/switch.py

```diff
@@ -77,16 +80,24 @@ def available(self) -> bool:
             and self.serial_number in self.coordinator.data["smart_plugs"]
         )
 
-    def turn_on(self, **kwargs: Any) -> None:
-        """Set smartplug status on."""
-        self.coordinator.verisure.set_smartplug_state(self.serial_number, True)
-        self._state = True
-        self._change_timestamp = monotonic()
-        self.schedule_update_ha_state()
+    async def async_turn_on(self, **kwargs: Any) -> None:
+        """Turn the smartplug on."""
+        await self.async_set_plug_state(STATE_ON)
```

Comment:
We don't have to pass a string here:

```suggestion
        await self.async_set_plug_state(True)
```